### PR TITLE
adapt wording within the code based on recent improvements

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: aggregate_test.go [hunk 1]
```diff
@@ -252,10 +252,11 @@ func Test_Aggregate(t *testing.T) {
 			ErrorMatcher: nil,
 		},
 
-		// Test 4 ensures depending bundles within the given capabilities are not
-		// bundled together within the aggregated state in case their dependency
-		// definitions do not meet their constraints. Thus the aggregated result
-		// should be empty because there is no proper bundle available.
+		// Test 4 ensures depending version bundles within the given list of version
```
Comment (MarcelMue) [inline aggregate_test.go]: Can you split this into two sentences? It runs on super long and is hard to understand currently.

File: aggregate_test.go [hunk 2]
```diff
@@ -328,9 +329,10 @@ func Test_Aggregate(t *testing.T) {
 			ErrorMatcher:           nil,
 		},
 
-		// Test 5 ensures when having a operator capability containing two bundles
-		// [a1,a2] and having another operator capability containing one bundle
-		// [b1], there should be two aggregated capabilities [[a1,b1],[a2,b1]].
+		// Test 5 ensures when having an operator's list of version bundles
```
Comment (MarcelMue) [inline aggregate_test.go]: Same here, maybe I am just a bit tired but the descriptions for Test 4,5&6 just run on in a single sentence.

File: aggregate_test.go (hunk 1 above)
Comment (xh3b4sd) [reply aggregate_test.go]: Yeah that was really weird. I tried to simplify it.

File: aggregate_test.go (hunk 2 above)
Comment (xh3b4sd) [reply aggregate_test.go]: Done.

File: aggregate_test.go (hunk 1 above)
Comment (MarcelMue) [reply aggregate_test.go]: Thanks, reads much better now!

**Pull request merged**